sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -51,21 +54,29 @@ export class ConfirmationOfEnrollmentController {
    * @returns student application list of an institution location
    */
   @HasLocationAccess("locationId")
-  @Get(":locationId/confirmation-of-enrollment")
+  @Get(
+    ":locationId/confirmation-of-enrollment/enrollmentPeriod/:enrollmentPeriod",
+  )
   async getCOESummary(
     @Param("locationId") locationId: number,
+    @Param("enrollmentPeriod") enrollmentPeriod: string,
```

Comment:
Can we pass the EnrollmentPeriod enum instead of string?

Reply:
Changed to enum from string.